Sources/Cocoa/Components/Picker/DrawerScreen/DrawerScreen.swift

```diff
@@ -113,6 +114,10 @@ final public class DrawerScreen: NSObject {
         }
     }
 
+    func setCaller(caller: Any?) {
+        drawerCaller = caller
+    }
+
```

Comment:
Are we able to get rid of this function? If I am not mistaken - the only access to `drawerCaller` is here in the class `DrawerScreen `

Reply:
Done!